configserver/src/main/java/com/yahoo/vespa/config/server/rpc/RpcServer.java

```diff
@@ -116,7 +116,8 @@ public RpcServer(ConfigserverConfig config, SuperModelRequestHandler superModelR
         this.metrics = metrics.getOrCreateMetricUpdater(Collections.<String, String>emptyMap());
         this.hostLivenessTracker = hostLivenessTracker;
         BlockingQueue<Runnable> workQueue = new LinkedBlockingQueue<>(config.maxgetconfigclients());
-        executorService = new ThreadPoolExecutor(config.numthreads(), config.numthreads(),
+        int numberOfRpcThreads = config.numRpcThreads() == 0 ? Runtime.getRuntime().availableProcessors() : config.numRpcThreads();
+        executorService = new ThreadPoolExecutor(numberOfRpcThreads, numberOfRpcThreads,
```

Comment:
I prefer parentheses around the condition. ? : is a bit tricky when it comes to evaluation order. Prefer explicit.